.eslintrc.js

```diff
@@ -502,6 +518,7 @@ module.exports = {
               SAFER_LODASH_SET_LODASH_HEADER,
               SAFER_LODASH_SET_DEFINITELYTYPED_HEADER,
               KBN_HANDLEBARS_HANDLEBARS_HEADER,
+              VENN_DIAGRAM_HEADER,
```

Comment:
__issue:__ it seems you forgot to forbid `VENN_DIAGRAM_HEADER` in `packages/kbn-handlebars/src/upstream/**/` (the rule below this one).